src/components/SidePanel/styles.module.scss

```diff
@@ -22,11 +22,17 @@
 }
 
 .content {
+  border-radius: $border-radius;
+  border: 1px solid $light-gray;
+  height: 100%;
   overflow-x: auto;
 }
 
+.borderlessContent {
+  border: 0;
```

Comment:
The shorthand for no border should be `none` (which applies to [`border-style`](https://www.w3schools.com/cssref/pr_border.asp)) instead of `0`. Using `0` technically works but may be quirky in some browsers.